templates/default/_ConfigurationFile.ini.erb

```diff
@@ -145,6 +149,9 @@ INSTANCENAME="<%= @instance_name %>"
 ; Agent account name
 
 AGTSVCACCOUNT="<%= @agent_account %>"
+<% unless @agent_account_pwd.nil? -%>
+AGTSVCPASSWORD="<%= @agent_acount_pwd %>"
```

Comment:
Having a password in this file is not good as this file is persistent on disk and represents a security concern.